src/wp-admin/includes/class-wp-plugin-install-list-table.php

```diff
@@ -828,4 +735,90 @@ public function display_rows() {
 			echo '</div></div>';
 		}
 	}
+
+	/**
+	 * Returns a notice containing a list of dependencies required by the plugin.
+	 *
+	 * @since 6.4.0
```

Comment:
Should be 6.5?

Reply:
Yea, needs fixing. This is essentially "copy" of https://github.com/WordPress/wordpress-develop/pull/3032 intended to be merged with the main PR after a bit of testing. The only changes are in /wp-admin/includes/class-wp-plugins-list-table.php.

Reply:
Fixed.